### PR TITLE
Always localize DeviceClass

### PR DESCRIPTION
**Pull request by infirit**
Re #1320

### CODE REVIEW

File: blueman/gui/manager/ManagerDeviceList.py [hunk 1]
```diff
@@ -275,13 +275,12 @@ def on_finished(fader: TreeRowFade) -> None:
         klass = get_minor_class(device['Class'])
         # Bluetooth >= 4 devices use Appearance property
         appearance = device["Appearance"]
-        if klass != "uncategorized" and klass != "unknown":
-            # get translated version
-            description = get_minor_class(device['Class'], True).capitalize()
-        elif klass == "unknown" and appearance:
+        if klass != _("Uncategorized") and klass != _("Unknown"):
+            description = get_minor_class(device['Class'])
```
Comment (cschramm) [inline blueman/gui/manager/ManagerDeviceList.py:279]:
```suggestion
            description = klass
```

**Review by cschramm — APPROVED** [review]

File: blueman/gui/manager/ManagerDeviceList.py [hunk 2]
```diff
@@ -246,8 +246,8 @@ def make_caption(name: str, klass: str, address: str) -> str:
     @staticmethod
     def get_device_class(device: Device) -> str:
         klass = get_minor_class(device['Class'])
-        if klass != "uncategorized":
-            return get_minor_class(device['Class'], True)
+        if klass != _("uncategorized"):
```
[the remark this answers is not in the document]
Comment (cschramm) [reply blueman/gui/manager/ManagerDeviceList.py]: I got confused somewhere... :sweat_smile:

[the remark this answers is not in the document]
Comment (infirit) [reply blueman/gui/manager/ManagerDeviceList.py]: Cool. I was wondering if I missed the point :+1:

File: blueman/gui/manager/ManagerDeviceList.py (hunk 1 above)
Comment (infirit) [reply blueman/gui/manager/ManagerDeviceList.py:279]: Wondering if you need to approve again, let's try to commit your suggestion :-)

Comment (cschramm) [reply blueman/gui/manager/ManagerDeviceList.py]: Of course. :smile:

**Review by cschramm — APPROVED** [review]

**Pull request merged**